include/z64.h

```diff
@@ -1433,7 +1433,7 @@ typedef struct {
     /* 0x2C */ s16*  owMinimapPosX;
     /* 0x30 */ s16*  owMinimapPosY;
     /* 0x34 */ s16 (*owCompassInfo)[4]; // [X scale, Y scale, X offset, Y offset]
-    /* 0x38 */ s16*  dgnMinimapTexIndexBase; // dungeon minimap texture index base
+    /* 0x38 */ s16*  dgnTexIndexBase; // dungeon texture index base
```

Comment:
technically not a part of your changes, but the dgn abbreviation is probably not needed in general? the name would be on the longer side for sure, but now that its shortened with your new changes it might be worth

I just dont see anyone use dgn ever, even tho its pretty easy to figure out what it means

Reply:
The abbreviation is used for many other names in there, including some pretty longer ones, so I kept it for consistency. Also this whole struct is already pretty verbose so I don't really think we should get rid of `dgn` (or `ow` for that matter)

Reply:
I dont think I agree, but it can be talked about later in context with those other variables separate from this set of changes

Reply:
Something I forgot to mention and might not be obvious here but all of these fields are arrays (with many 2d or even 3d arrays), so short names are more important than usual